src/components/MetadataForm/MetadataForm.tsx

```diff
@@ -7,6 +7,8 @@ import { OnChangeCallback } from '../../types';
 
 export type Props = Omit<ComponentProps<typeof JsonForms>, 'onChange'> & {
 	onChange: OnChangeCallback;
+	headerTitle?: string | null;
```

Comment:
I'd personally prefer naming it just `heading`

Reply:
Changed